incubator/minecraft/templates/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rcon-password: {{ default "" .Values.MinecraftServer.Rcon.Password | b64enc | quote }}
```

Comment:
I agree with @gtaylor that although the implementation is not perfect we should not be banking on a feature that is not ever intended for keeping secrets. We hope that K8s will make this better over time, best to attach ourselves to the proper feature.